main.tf

```diff
@@ -24,15 +24,17 @@ resource "datadog_monitor" "generic_datadog_monitor" {
   })
 
   tags = concat(
-  [
-    "terraform:true",
-    "env:${var.env}",
-    "service:${var.service}",
-    "severity:${var.severity}",
-  ],
-  var.additional_tags
+    [
```

Comment:
@obeleh: Us fighting over the indentation in this case is going to get old really quick. I suggest we use what `terraform fmt` does, because that indents it the way I like it. 🙂

Reply:
I'm not fighting you. I was using `terraform fmt` afaik

Reply:
I'll check if it was PyCharm